pkg/crypto/tinkcrypto/primitive/composite/keyio/composite_key_pubkey_to_handle.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package keyio
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/tink/go/aead"
+	"github.com/google/tink/go/insecurecleartextkeyset"
+	"github.com/google/tink/go/keyset"
+	commonpb "github.com/google/tink/go/proto/common_go_proto"
+	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite"
+	ecdh1pupb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/ecdh1pu_aead_go_proto"
+	ecdhespb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/ecdhes_aead_go_proto"
+)
+
+// PrimitiveCompositeType represents the type of composite primitive
+type PrimitiveCompositeType int
+
+const (
+	// UNKNOWN composite primitive type
+	UNKNOWN PrimitiveCompositeType = iota
+
+	// ECDH1PU composite primitive type
+	ECDH1PU
+
+	// ECDHES composite primitive type
+	ECDHES
+)
+
+// publicKeyBytesToHandle for reading pubKey and get keyset.Handle of either ECDHES or ECDH1PU composite public key type
+func publicKeyBytesToHandle(pubKey []byte, compositeType PrimitiveCompositeType) (*keyset.Handle, error) {
```

Comment:
only used within tests

Reply:
Good catch, this functionally was never used by ECDH-ES originally. Similar function exists in localkms's public key reader to convert a Public key to a Tink key handle. It should be updated to support ECDH-ES/ECDH-1PU keys as well (if needed in the future).

I will remove this function.

Reply:
function is now removed